### DISCUSSION TITLE
Recent Search/Query History

### DISCUSSION THREAD

**Discussion by ryanbuening** [datalust/seq-tickets#2310] [Ideas]
Is there a way to see recent queries in Seq? I find this feature to be useful in other logging systems. I think integrating something directly into the filter/search bar would be great.

Below is an example from Rapid7.

![image](https://github.com/user-attachments/assets/ce6621f8-260f-44f6-8656-d6b036d04018)

**Comment by nblumhardt** [comment]
Hi Ryan,

In the right hand bar you should see a "History" tab - is this the one you're looking for?

Cheers!

**Reply by ryanbuening** [reply]
TIL. 👍
